openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -1052,7 +961,7 @@ def build_jwt_headers(self, user):
         return headers
 
     def build_post(self, username):
-        return {'user': username}
+        return {'username': username}
```

Comment:
I 👍 this standard of "username" being the actual username - and "user" being the Django user model.